UI/js/kitchen.js

```diff
@@ -0,0 +1,11 @@
+const orderDetail = (e) => {
```

Comment:
Let's use a descriptive variable name. Change `e` to `event`

Reply:
I am planning of doing that when refactoring the codes